src/mlpack/methods/dbscan/random_point_selection.hpp

```diff
@@ -45,11 +50,15 @@ class RandomPointSelection
         ++found;
 
       if (found > index)
+      {
+        unvisited[i]=false;
```

Comment:
Here we should be careful with the style: `unvisited[i] = false`.

Reply:
@rcurtin Thanks for taking the time and going through my code.
I will go through the style and contributions guide and take better care of this next time :+1: .